src/main/java/appeng/helpers/DualityInterface.java

```diff
@@ -1269,7 +1273,8 @@ public void setPriority( final int newValue )
 
 	public boolean hasCapability( Capability<?> capabilityClass, EnumFacing facing )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "facing". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1172)